### DISCUSSION TITLE
Do you support PHP8.0.x or better?

### DISCUSSION THREAD

**Discussion by tooliedotter** [shlinkio/discussions#32] [Help wanted] [answered]
I'm about to move my websites (including my self-hosted link-shortening site) to a new server with PHP 8.0.x and MariaDB 8. Does your software support those libraries? Thanks!

**Comment by acelaya** [comment, marked as answer]
I guess you talk about Shlink and not shlink-web-client.

If so, yes, it supports both PHP 8 and MariaDB https://shlink.io/documentation/install-dist-file/

**Comment by tooliedotter** [comment]
Perfect, thank you! I will follow up and try an installation.

Toolie® Garner
***@***.******@***.***>
Mobile Office: +1-425-830-0740
https://www.remoteleadershipsuccess.com<https://www.remoteleadershipsuccess.com/>
https://www.linkedin.com/in/toolie
